com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/topology.py

```diff
@@ -40,8 +49,13 @@ def __init__(self, name, files=None):
           self.opnamespace = "com.ibm.streamsx.topology.functional.python2"
         else:
           raise ValueError("Python version not supported.")
-        self.dependent_packages = topologypackages.TopologyPackages()
-        self.graph = graph.SPLGraph(name, self.dependent_packages)
+        self._conda_packages_to_exclude = set(["numpy", "scipy", "matplotlib", "h5py"])
```

Comment:
Just a nit, but why do you need to define `_conda_packages_to_exclude` as a field of Topology? Couldn't you just set `self.exclude_packages` to `set(["numpy", "scipy", "matplotlib", "h5py"])` on line 55?